.github/workflows/test-forked.yml

```diff
@@ -19,13 +19,6 @@ jobs:
       - name: allowed message
         run: echo "Allowed to run"
 
-  unit-tests-forked:
```

Comment:
Shouldn't we have a similar flow for forked contributions? run the lint + unit test without restriction and run the other tests once the PR is labeled.

Reply:
I can look into that. Not very familiar with the forked vs not forked thing.

Reply:
done